packages/safe-tools-client/app/components/block-explorer-button/index.gts

```diff
@@ -38,9 +42,17 @@ export default class BlockExplorerButton extends Component<Signature> {
       @as="anchor"
       @size="extra-small"
       @kind={{@kind}}
-      href={{this.blockExplorerUrl}} 
+      @href={{this.blockExplorerUrl}}
+      @disabled={{not @transactionHash}}
       target="_blank"
       rel="noopener"
+      class={{cn "block-explorer-button" block-explorer-button--disabled=(not @transactionHash)}}
+      style={{cssVar
+        boxel-button-color="transparent"
+        boxel-button-border=(if @transactionHash "1px solid var(--boxel-purple-300)" "1px solid var(--boxel-purple-200)")
```

Comment:
I'm a bit confused why we have styling in two places - in the CSS file and also using the `cssVar` helper